app/Models/Recipe.php

```diff
@@ -23,13 +23,47 @@ public function cookbook_rel()
     {
         return $this->belongsTo('App\Models\Cookbook', 'cookbook', 'slug');
     }
+    
+    public function comments()
+    {
+        return $this->hasMany('App\Models\Comment', 'recipe_tracking_nr', 'tracking_nr');
+    }
 
     // Global scopes
 
     use HasVisibilities;
 
     // Helpers
     
+    public function getRating($output = "average")
+    {
+        $ratings = $this->comments();
+        $count = $ratings->whereNotNull('rating')->count();
+        $average = number_format($ratings->avg('rating'),1);
+        switch($output){
+            case 'average':
+                return $average;
+                break;
```

Comment:
You don't need an additional `break` statement after a `return`, since the function has already exited.